build/repo.targets

```diff
@@ -106,19 +106,11 @@
       Condition="'$(CloneRepositoryCommit)'!=''" />
   </Target>
 
-  <Target Name="_CopyKorebuild" AfterTargets="_CloneRepository">
-    <ItemGroup>
-      <_CopyItems Include="$(RepositoryRoot).build\**\*.*" />
-    </ItemGroup>
-    
-    <Message Text="Copying Korebuild from Universe to repository $(_CloneRepositoryRoot)$(CloneRepository)" />
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_CloneRepositoryRoot)$(CloneRepository)\.build\%(RecursiveDir)" />
-  </Target>
-  
   <Target Name="BuildRepositories"
      DependsOnTargets="_FilterRepositories;_FindDotNetPath;_GenerateRestoreGraphSpecs;_GenerateBuildGraph;_UpdateNuGetConfig;_CreateRepositoriesListWithCommits">
 
     <PropertyGroup>
+      <_RepoDir>$(_CloneRepositoryRoot)%(Repository.Identity)</_RepoDir>
```

Comment:
You are right. While testing, I took a shortcut by setting the environment variable to include a single repsository which is why it worked, but it fails when there are multiple. I moved this script to a different location now.